docs/sources/panels-visualizations/configure-thresholds/index.md

```diff
@@ -20,66 +20,101 @@ weight: 100
+| As lines (dashed)                    | {{< figure max-width="500px" src="/media/docs/grafana/panels-visualizations/screenshot-thresholds-dashed-lines-v10.4.png" alt="Visualization with threshold as a dashed line" >}}                    |
+| As filled regions                    | {{< figure max-width="500px" src="/media/docs/grafana/panels-visualizations/screenshot-thresholds-regions-v10.4.png" alt="Visualization with threshold as a region" >}}                              |
+| As filled regions and lines          | {{< figure max-width="500px" src="/media/docs/grafana/panels-visualizations/screenshot-thresholds-lines-regions-v10.4.png" alt="Visualization with threshold as a region and line" >}}               |
+| As filled regions and lines (dashed) | {{< figure max-width="500px" src="/media/docs/grafana/panels-visualizations/screenshot-thresholds-dashed-lines-regions-v10.4.png" alt="Visualization with threshold as a region and dashed line" >}} |
 
-1. Edit the panel to which you want to add a threshold.
-
-1. In the options side pane, locate the **Thresholds** section and click **+ Add threshold**.
-
-1. Select a threshold color, number, and mode.
-   Threshold mode applies to all thresholds on this panel.
+## Add a threshold
 
-1. For a time-series panel, select a **Show thresholds** option.
+You can add as many thresholds to a visualization as you want. Grafana automatically sorts thresholds values from highest to lowest.
 
-Delete a threshold when it is no longer needed. When you delete a threshold, the system removes the threshold from all visualizations that include the threshold.
+1. Navigate to the panel you want to update.
+1. Hover over any part of the panel you want to work on to display the menu on the top right corner.
+1. Click the menu and select **Edit**.
+1. Scroll to the **Thresholds** section or enter `thresholds` in the search bar at the top of the panel edit pane.
+1. Click **+ Add threshold**.
+1. Enter a new threshold value or use the up and down arrows at the right side of the field to increase or decrease the value incrementally.
+1. Click the colored circle to the left of the threshold value to open the color picker, where you can update the threshold color.
+1. Under **Thresholds mode**, select either **Absolute** or **Percentage**.
+1. Under **Show thresholds**, set how the threshold is displayed or turn it off.
 
 To delete a threshold, navigate to the panel that contains the threshold and click the trash icon next to the threshold you want to remove.
 
 ## Add a threshold to a legacy graph panel
 
+{{< admonition type="caution" >}}
+Starting with Grafana v11, the legacy graph panel will be deprecated along with all other Angular panel plugins. It will be removed in a future release. For more information, refer to [Angular support deprecation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/developers/angular_deprecation/).
```

Comment:
@nmarrs - Is this note about the deprecation accurate? I'm using boilerplate language so I'm not sure if the second sentence applies here ("It will be removed in a future release.").

Reply:
I think we can get rid of the second sentence 👍